### ISSUE TITLE
Rebooting log messages no longer appearing

### ISSUE THREAD

**Issue by neryskistner** [cpanel/elevate#70]
```
* 08-19:17:38 (1056) [INFO] ******************************************************************************************
* 08-19:17:38 (1057) [INFO] *
* 08-19:17:38 (1058) [INFO] * Rebooting into stage 3 of 5
* 08-19:17:38 (1059) [INFO] *
* 08-19:17:38 (1060) [INFO] ******************************************************************************************
* 08-19:17:38 (2848) [INFO] Running: /usr/sbin/reboot now
```

This log message no longer appears in when running elevate, and the user's terminal session ends when it reboots without telling them why that happened.

**Comment by toddr** [comment]
I'm pretty sure this is because the output is buffering. I think we need to cut out tee.

**Comment by sloanebernstein** [comment]
This may be addressed by reverting #67 in #77.

**Comment by toddr** [comment]
This is no longer an issue and has been corrected.
